src/server/getters/database.ts

```diff
@@ -68,17 +74,18 @@ export function reshapeMarketsRowToUIMarketInfo(row: MarketsRowWithCreationTime,
     details: row.longDescription,
     scalarDenomination: row.scalarDenomination,
     designatedReporter: row.designatedReporter,
-    designatedReportStake: row.designatedReportStake,
+    designatedReportStake: row.designatedReportStake!,
     resolutionSource: row.resolutionSource,
```

Comment:
Non-null assertion. Basically says "This property is NOT null/undefined in this context".

Useful since designatedReportStake is optional on the `row` type.